From: Borislav Petkov <bp@suse.de>
Date: Sat, 18 Jan 2014 12:48:15 +0100
Subject: x86/efi: Dump the EFI page table
Patch-mainline: Never, SUSE-Xen specific
References: fate#315017

This is very useful for debugging issues with the recently added
pagetable switching code for EFI virtual mode.

Signed-off-by: Borislav Petkov <bp@suse.de>
Tested-by: Toshi Kani <toshi.kani@hp.com>
Signed-off-by: Matt Fleming <matt.fleming@intel.com>
Automatically created from "patches.arch/x86-efi-dump-the-efi-page-table.patch" by xen-port-patches.py

--- sle12.orig/arch/x86/Kconfig.debug	2014-03-19 15:08:37.000000000 +0100
+++ sle12/arch/x86/Kconfig.debug	2014-03-19 15:09:23.000000000 +0100
@@ -84,7 +84,7 @@ config X86_PTDUMP
 
 config EFI_PGT_DUMP
 	bool "Dump the EFI pagetable"
-	depends on EFI && X86_PTDUMP
+	depends on EFI && X86_PTDUMP && !XEN
 	---help---
 	  Enable this if you want to dump the EFI page table before
 	  enabling virtual mode. This can be used to debug miscellaneous
